### PR TITLE
[SUPLY-1170] - adding gcp pods to the backend

### PR DESCRIPTION
**Pull request by jeffnoehren**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,7 @@
+source "https://rubygems.pkg.github.com/apartmentlist"
 source 'https://rubygems.org'
 
+
 # Specify your gem's dependencies in bulk-processor.gemspec
+# gem 'gcp_job_manager', github: 'apartmentlist/gcp_job_manager'
```
Comment (abezzub) [inline Gemfile:6]: should this be removed?

File: lib/bulk_processor/back_end/gcp_pods.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    class GcpPods
+      def initialize(processor_class:, payload:, key:)
+        @processor_class = processor_class.name
+        @payload = PayloadSerializer.serialize(payload)
+        @key = key
+      end
+
+      def start
+        args = [processor_class, payload, key]
+        GcpJobManager.create_and_deploy_job('start-bulk-processor', args)
```
Comment (abezzub) [inline lib/bulk_processor/back_end/gcp_pods.rb:14]: were you saying we would need to copy/paste these jobs in every service's config? I think there might be a way to make them added as part of this gem.

File: Gemfile (hunk 1 above)
Comment (jeffnoehren) [reply Gemfile:6]: 👍

File: lib/bulk_processor/back_end/gcp_pods.rb (hunk 2 above)
Comment (jeffnoehren) [reply lib/bulk_processor/back_end/gcp_pods.rb:14]: 👍

**Pull request merged**